lib/core/src/lib/dynamic-chip-list/dynamic-chip-list.component.scss

```diff
@@ -6,7 +6,6 @@
     padding-bottom: 12px;
 
     .adf-dynamic-chip-list-view-more-button {
-        color: var(--adf-theme-foreground-text-color-054);
```

Comment:
The button should have no color at all?

Reply:
Also it is actually touching the last chip, is it okay for us? 
<img width="632" alt="Zrzut ekranu 2024-06-7 o 12 43 09" src="https://github.com/Alfresco/alfresco-ng2-components/assets/113341662/21ac13f2-4fef-44ce-a8d1-393bbcc11e89">

Reply:
1. with or without this color it looks the same, if we would want to add some color, I would be into using general theming option instead manual